### PR TITLE
Merging Migration branch into main

### PR DESCRIPTION
**Pull request by HenrijetteOest**
Updated with:
-onion structure: 
* ChirpCore
* ChirpInfrastrure
* ChirpWeb
-Migration
-Database is embedded in Program.cs
new sln file

Co-authored-by: Laura <llsa@itu.dk>
Co-authored-by: Maria <arnv@itu.dk>
Co-authored-by: Henriette <hoes@itu.dk>

### CODE REVIEW

File: src/ChirpCore/ChirpCore.csproj [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
     <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="8.0.10">
       <IncludeAssets>runtime; build; native; contentfiles;
       Analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
 
-    <PackageReference Include="Microsoft.Data.Sqlite" Version="8.0.10" />
-    <EmbeddedResource Include="data/**/*" />
-  </ItemGroup>
+		<PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="8.0.8">
```
Comment (tomat-suppe) [inline src/ChirpCore/ChirpCore.csproj:26]: Just leaving comment for later in case we get problems with EFCore - Tools 8.0.10 like our EFCore package?

**Review by Thyme-Cat — APPROVED** [review]
Many cool changes

**Review by tomat-suppe — APPROVED** [review]
looks good, lets get this website

**Review by JaSNOHo — APPROVED** [review]
lgtm

**Pull request merged**